diff --git a/src/modules/wlr/taskbar.cpp b/src/modules/wlr/taskbar.cpp
--- a/src/modules/wlr/taskbar.cpp
+++ b/src/modules/wlr/taskbar.cpp
@@ -102,1 +102,10 @@ Task::Task(const waybar::Bar& bar, const Json::Value& config, Taskbar* tbar,
   button.add(content_);
+  if (config_["truncate"].isBool() && config_["truncate"].asBool()) {
+    text_before_.set_single_line_mode(true);
+    text_before_.set_ellipsize(Pango::ELLIPSIZE_END);
+    text_before_.set_line_wrap(false);
+
+    text_after_.set_single_line_mode(true);
+    text_after_.set_ellipsize(Pango::ELLIPSIZE_END);
+    text_after_.set_line_wrap(false);
+  }
